userdocs/diagnostics/actor-isolated-call.md

```diff
@@ -0,0 +1,46 @@
+# Calling an actor-isolated method from a synchronous nonisolated context
+
+Calls to actor-isolated methods from outside the actor must be done asynchronously. Otherwise, access to actor state can happen concurrently and lead to data races. These rules also apply to global actors like the main actor.
+
+For example:
+
+```swift
+@MainActor
+class MyModel {
+  func update() { ... }
+}
+
+func runUpdate(model: MyModel) {
+  mode.update()
+}
+```
+
+Building the above code produces an error about calling a `@MainActor`-isolated method from outside the actor:
```

Comment:
```suggestion
Building the above code produces an error about calling a method isolated to the main actor from outside the actor:
```

Reply:
I'm inclined to change all occurrences of "`@MainActor`-isolated" in prose to "main actor isolated". Thoughts?

Reply:
Very much in favor. It removes the jargon feel while keeping the concrete relevant detail in place. Using the representation (or type) in the name can be useful in some situations, but I don't think there's variation enough to warrant the using the `@MainActor` declaration as part of word in these cases.

Reply:
(Keeping the declaration mixed in also makes it harder to translate down the road, for what that's worth)